### ISSUE TITLE
[TS] $traceurRuntime is not defined

### ISSUE THREAD

**Issue by SamVerschueren** [angular/angular.io#102]
When following the [first step](https://angular.io/docs/js/latest/guide/setup.html) of the step-by-step guide, this is the code of the TS example

```html
<!DOCTYPE html>
<html>
  <head>
    <link rel="stylesheet" href="style.css">
    <script src="https://jspm.io/system@0.16.js"></script>
    <script src="https://code.angularjs.org/2.0.0-alpha.22/angular2.dev.js"></script>
  </head>
  <body>
    <my-app></my-app>
    <script>
      System.import('main');
    </script>
  </body>
</html>
```

If I execute this example, I get the error about the $traceurRuntime not being defined. If I hadd the traceurRuntime script, it works correctly.

```html
<!DOCTYPE html>
<html>
  <head>
    <script src="https://github.jspm.io/jmcriffey/bower-traceur-runtime@0.0.87/traceur-runtime.js"></script>
    <script src="https://jspm.io/system@0.16.js"></script>
    <script src="https://code.angularjs.org/2.0.0-alpha.22/angular2.dev.js"></script>
  </head>
  <body>
    <my-app></my-app>
    <script>
      System.import('main');
    </script>
  </body>
</html>
```

**Comment by alexwolfe** [comment]
@davideast @rkirov can one of you look at this? Thanks.

**Comment by alexeagle** [comment]
fixed in #143
